encrypted-media-respec.html

```diff
@@ -2493,13 +2493,25 @@
             <li><p>Let <var>session</var> be the associated <a>MediaKeySession</a> object.</p></li>
             <li>
               <p>
-                If <var>session</var>'s <var>session type</var> is <a def-id="persistent-usage-record-session"></a>,
-                store <var>session</var>'s <var>record of key usage</var>, if it exists.
+                If <var>session</var>'s <var>session type</var> is <a def-id="persistent-usage-record-session"></a>, execute the following steps:
```

Comment:
As per [this comment](https://github.com/w3c/encrypted-media/issues/225#issuecomment-224280044), it makes no difference: "in parallel" means only that page-visible effects don't happen immediately, but there are no page-visible effects. So, instead, we have the note below.